install_config/monitoring/configuring-monitoring.adoc

```diff
@@ -28,4 +28,19 @@ The OpenShift Ansible `openshift_cluster_monitoring_operator` role configures an
 |`openshift_cluster_monitoring_operator_alertmanager_config`
 | Configures Alertmanager. See xref:../install_config/cluster_monitoring.adoc#configuring-alertmanager[Configuring Alertmanager]
 
+|`openshift_cluster_monitoring_operator_prometheus_storage_enabled`
+| Enable persistent storage of Prometheus time-series data.
+
+|`openshift_cluster_monitoring_operator_alertmanager_storage_enabled`
+| Enable persistent storage of Alertmanager notifications and silences.
+
 |===
+
+== Configuring persistent storage
```

Comment:
I would probably highlight that you don't need to set both if you want to have persistent storage. Depending on your use case, you would set one or the other, or both. 

I would also explain how to increase the storage size and probably a paragraph on dynamic provisioning vs static provisioning.

I would almost copy things from here: https://docs.openshift.com/container-platform/3.10/install_config/cluster_metrics.html#metrics-persistent-storage

Like the first paragraph should also be the first paragraph in this section. It could look like the following:

```
Running OpenShift Container Platform cluster metrics with persistent storage means that your metrics are stored to a [persistent volume](https://docs.openshift.com/container-platform/3.10/architecture/additional_concepts/storage.html#persistent-volumes) and are able to survive a pod being restarted or recreated. This is ideal if you require your metrics or alerting data to be guarded from data loss. For production environments it is highly recommended to configure persistent storage.
```

After that you can talk about that persistent storage for Prometheus and Alertmanager is disabled by default and to enable one or the other, or both; you need to set these variables inside your inventory file, bla bla. :)

After that you can start with something like that to ensure that we include how you configure the size:

```
The size requirement of the storage is dependent on the number of pods. It is the administrator’s responsibility to ensure that the size requirements are sufficient for their setup to ensure that the disk does not become full. The size of the persisted volume claim for Prometheus and Alertmanager is specified in the following Ansible variables:

* openshift_cluster_monitoring_operator_prometheus_storage_capacity (default: 50Gi)
* openshift_cluster_monitoring_operator_alertmanager_storage_capacity (default: 2Gi)
```

After that you can put a short paragraph on dynamic provisioning. There is one inside the cluster metrics chapter as well.

Maybe we can also go into a bit static provisioning just mentioning that for each replica, you need to have a pv ready for the pvc to claim. That means, Prometheus has 2 replicas and Alertmanager has 3.

Reply:
Implemented